op-e2e/system_adminrpc_test.go

```diff
@@ -66,10 +68,11 @@ func TestStopStartSequencer(t *testing.T) {
 	require.NoError(t, err)
 	require.True(t, active, "sequencer should be active again")
 
-	blockBefore = latestBlock(t, l2Seq)
-	time.Sleep(time.Duration(cfg.DeployConfig.L2BlockTime+1) * time.Second)
-	blockAfter = latestBlock(t, l2Seq)
-	require.Greater(t, blockAfter, blockBefore, "Chain did not advance after starting sequencer")
+	require.NoError(
+		t,
+		wait.ForNextBlockWithTimeout(ctx, l2Seq, time.Duration(cfg.DeployConfig.L2BlockTime+1)*2*time.Second),
```

Comment:
I think to reduce intermittency we probably need to increase the length of the timeout here

Reply:
Wound up simplifying things a lot and just modifying `ForNextBlock` to always apply a 60 second timeout.  That will be long enough to cover any sensible block time and means we have a timeout in place for all tests to avoid them hanging.  Test code is then kept really simple and declarative with the details being handled by the helper code.